src/scripts/main.js

```diff
@@ -22,6 +22,23 @@ const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
   // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
```

Comment:
Code Quality Issue: You are using a `for...in` loop to iterate over the properties of an object. While this is not wrong, it's generally recommended to use `Object.keys()` with a `forEach` or `for...of` loop to iterate over an object's properties. This is because `for...in` will also enumerate properties in the prototype chain, which can lead to unexpected behavior if the object's prototype has enumerable properties.